### PR TITLE
Pass -g, -r, -m to htrun

### PR DESCRIPTION
**Pull request by studavekar**
Fix for issue : https://github.com/ARMmbed/greentea/issues/260 

pass additional parameter which would be useful for htrun.

### CODE REVIEW

**Comment by theotherjimmy** [comment]
@studavekar Does this need to be released soon?

**Comment by studavekar** [comment]
> @studavekar Does this need to be released soon?

we are working off master.  If its merged in we should be good.

File: mbed_greentea/mbed_test_api.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def run_host_test(image_path,
     @param duration Test case timeout
     @param micro Mbed-enabled device name
     @param reset Reset type
-    @param reset_tout Reset timeout (sec)
+    @param forced_reset_timeout Reset timeout (sec)
```
Comment (bridadan) [inline mbed_greentea/mbed_test_api.py]: Can you also remove the `reset_tout` parameter from the function definition?

Comment (studavekar) [reply mbed_greentea/mbed_test_api.py]: done

**Comment by cmonr** [comment]
@theotherjimmy Good to merge?

**Pull request merged**